pkg/codegen/pcl/functions_test.go

```diff
@@ -113,6 +115,69 @@ func TestTryWithCorrectArguments(t *testing.T) {
 	// Assert.
 	assert.NotNil(t, program, "The program binds")
 	assert.NoError(t, err)
+
+	// Assert that the type of the variable is a plain number.
+	assert.Equal(t, len(program.Nodes), 1, "there is one node")
+	localVariable, ok := program.Nodes[0].(*pcl.LocalVariable)
+	assert.True(t, ok, "first node is a local variable variable")
+	variableType := localVariable.Type()
+	num := func(i int) *model.ConstType {
+		return model.NewConstType(model.NumberType, cty.NumberVal(new(big.Float).SetInt64(int64(i)).SetPrec(512)))
+	}
+	expectedType := model.NewUnionType(num(1), num(2), num(3))
+	assert.Equal(t, expectedType, variableType, "the type is a plain union")
+}
+
+// Tests that the PCL `try` intrinsic function binds when correctly passed any non-zero number of arguments.
```

Comment:
Maybe tweak this comment since I'm guessing it's specifically about outputs here. And perhaps the one below too?

Reply:
ack